queries/models/dashboard_monitoramento_interno/schema.yml

```
@@ -24,5 +24,31 @@ models:
         description: "{{ doc('id_veiculo') }}"
       - name: indicador_regularidade_temperatura
         description: "Indicador de regularidade da temperatura da viagem"
+      - name: operadora
+        description: "{{ doc('operadora') }}"
+models:
```

YAMLlint (1.38.0)

[error] duplication of key "models" in mapping

(key-duplicates)